code/model/SiteTree.php

```diff
@@ -1411,29 +1399,23 @@ public function MetaTags($includeTitle = true) {
 		return $tags;
 	}
 
-
 	/**
-	 * Returns the object that contains the content that a user would
-	 * associate with this page.
+	 * Returns the object that contains the content that a user would associate with this page.
 	 *
-	 * Ordinarily, this is just the page itself, but for example on
-	 * RedirectorPages or VirtualPages ContentSource() will return the page
-	 * that is linked to.
+	 * Ordinarily, this is just the page itself, but for example on RedirectorPages or VirtualPages ContentSource() will
+	 * return the page that is linked to.
 	 *
-	 * @return SiteTree The content source.
+	 * @return SiteTree The content source
```

Comment:
How about `@return static`? That would ensure the correct type hinting for subclasses (considering the function returns `$this`). `@return self` implies `SiteTree`. In fact, I usually document it as `@return $this` which works well in PhpStorm but I'm not sure if it's standard.